src/components/Sidebar.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { storeCustomers } from "../data";
+import { CustomerConsumer } from "./CustomerContext";
+
+export default function Sidebar() {
+  return (
+    <CustomerConsumer>
+      {(ctx) => {
+        function getSelectedCustomer() {
+         let selectedCustomer = document.getElementById("storeCustomers")
+         ctx.setCustomer(selectedCustomer.value)
+        }
+        return (
+          <div className="msb" id="msb">
+            <nav className="navbar navbar-default" role="navigation">
+              <div className="navbar-header">
+                <div className="brand-wrapper">
+                  <div className="brand-name-wrapper">
+                    <p className="navbar-brand text-center">Control panel</p>
+                  </div>
+                </div>
+              </div>
+              <div className="side-menu-container">
```

Comment:
not necessary now, but you might want to extract this selector to a separate React component. This file will grow rapidly in future.